src/services/dispatchers/dispatchers/types.ts

```diff
@@ -52,7 +52,27 @@ export interface DispatchersService {
    * @param options - Additional options for the request
    * @returns The created Dispatcher {@link Dispatcher}
    */
-  create(requestBody: Dispatcher, options?: OptionsBase): Promise<Dispatcher>;
+  create(
+    requestBody: CreateDispatcherBody,
+    options?: OptionsBase
+  ): Promise<Dispatcher>;
+
+  /**
+   * ## Update a Dispatcher
+   *
+   * **Global Permissions:**
+   * - `UPDATE_DISPATCHERS` - Allows a user to update Dispatchers
+   *
+   * @param dispatcherId {@link ObjectId} The id of the Dispatcher to be updated
+   * @param requestBody The data used to update the Dispatcher
```

Comment:
(tiny) Can you add a link to the types here too for the body and options, I think we normally include them too inline? 🤔